src/utils/generate_cxx_code.cpp

```diff
@@ -14,16 +14,24 @@
 #error "no config"
 #endif
 
-#include <unordered_map>
-#include "sbml_utils.hpp"
-#include <cstdio>
-
-#if defined(WCS_HAS_SBML)
+#include "utils/generate_cxx_code.hpp"
+#include "utils/exception.hpp"
+#include "utils/file.hpp"
 #include <iostream>
+#include <fstream>
+#include <sstream>
+#include <stdlib.h> //system
+#include "wcs_types.hpp"
+#include <set>
+#include <regex>
+
 
-#include <sbml/SBMLTypes.h>
-#include <sbml/common/extern.h>
+#include <string>
+#include <filesystem>
```

Comment:
remove this since you include 'utils/file.hpp', which chooses 'filesystem' or 'boost/filesystem' depending on the availability of the library in the platform.